lib/src/actions/user/userproviders/none.rs

```diff
@@ -0,0 +1,22 @@
+use super::UserProvider;
+use crate::steps::Step;
+use crate::{actions::user::UserVariant, atoms::command::Exec};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct NoneUserProvider {}
+
+impl UserProvider for NoneUserProvider {
+    fn add_user(&self, _user: &UserVariant) -> Vec<Step> {
```

Comment:
This can return an empty vec! of steps and can we ensure it adds a warning with `use tracing::warn` and `warn!("")`?

Reply:
Good catch, I completely forgot about the None provider just having some test code in it.